hewr/R/process_state_forecast.R

```diff
@@ -1,21 +1,104 @@
+#' Combine training and evaluation data for
+#' postprocessing.
+#'
+#' @param train_dat Training data, as a [`tibble`][tibble::tibble()].
+#' @param eval_dat Evaluation data, as a [`tibble`][tibble::tibble()].
+#' @param disease_name Name of the target disease.
+#' One of `"COVID-19"` or `"Influenza"`.
+#' @return The combined data, as a [`tibble`][tibble::tibble()].
+#' @export
+combine_training_and_eval_data <- function(train_dat,
+                                           eval_dat,
+                                           disease_name) {
+  combined_dat <- dplyr::bind_rows(
+    train_dat |>
+      dplyr::filter(.data$data_type == "train"),
```

Comment:
Now that we have sorted out a way to get eval data, we should probably stop having non-training data in the `train_dat`. Would require changes to https://github.com/CDCgov/pyrenew-hew/blob/main/pipelines/prep_data.py

Reply:
Created #207